cli/cmd/compliance_aws.go

```diff
@@ -223,6 +232,7 @@ To show recommendation details and affected resources for a recommendation id:
 		Long:    `Run a compliance assessment for the provided AWS account.`,
 		Args:    cobra.ExactArgs(1),
 		RunE: func(_ *cobra.Command, args []string) error {
+			// Todo(v2): replace with v2
```

Comment:
Guess this would be replaced by onDemand ingest?

Reply:
I think I'll just remove this comment. We'll know where this change is when we go to replace it.